src/components/NewProject.js

```diff
@@ -0,0 +1,12 @@
+import React, { Fragment } from 'react';
+import { Header } from 'semantic-ui-react';
+
+export default function NewProject() {
+  return (
+    <Fragment>
```

Comment:
we already have a way to create new projects, please check out `CreateProjectPage`

Reply:
maybe the issue was not clear enough? let me make update to explicitly state this

Reply:
Got it. Just seeing the CreateProjectPage now. I will update my fork and make the requested changes shortly.

Reply:
great @ALau2088, thanks for making the requested changes! :smile_cat:
just a slight change to the tests is needed see :point_down: